app/src/main/java/org/p2p/wallet/sell/ui/lock/SellLockFragment.kt

```diff
@@ -0,0 +1,27 @@
+package org.p2p.wallet.sell.ui.lock
+
+import android.os.Bundle
+import android.view.View
+import org.koin.android.ext.android.inject
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BaseMvpFragment
+import org.p2p.wallet.databinding.FragmentSellLockBinding
+import org.p2p.wallet.utils.viewbinding.viewBinding
+
+class SellLockFragment :
+    BaseMvpFragment<SellLockContract.View, SellLockContract.Presenter>(R.layout.fragment_sell_lock),
+    SellLockContract.View {
+
+    companion object {
+        fun create() = SellLockFragment()
+    }
+
+    override val presenter: SellLockContract.Presenter by inject()
+    private val binding: FragmentSellLockBinding by viewBinding()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+        }
```

Comment:
why it's empty?
If your PR is not ready, please mark it with a label

Reply:
Yeah design was changed 3 times, while i was developing this, we will discuss about this today on standup
That's why PR was not finished.